_pkgdown.yml

```diff
@@ -242,6 +242,8 @@ navbar:
     - text: "------------------"
     - text: "Change log"
       href: news/index.html
+  - text: Extensions
+    href: https://www.ggplot2-exts.org
```

Comment:
Should it link directly to http://www.ggplot2-exts.org/gallery/ ?

Reply:
yeah, that is probably a more inviting landing page...